pkg/test/masteruserrecord/master_user_record.go

```diff
@@ -139,12 +133,11 @@ func Namespace(nsType, revision string) UaInMurModifier {
 		for uaIndex, ua := range mur.Spec.UserAccounts {
 			if ua.TargetCluster == targetCluster {
 				for nsIndex, ns := range mur.Spec.UserAccounts[uaIndex].Spec.NSTemplateSet.Namespaces {
-					if ns.Type == nsType {
-						mur.Spec.UserAccounts[uaIndex].Spec.NSTemplateSet.Namespaces[nsIndex].Revision = revision
+					if strings.Contains(ns.TemplateRef, nsType) {
 						mur.Spec.UserAccounts[uaIndex].Spec.NSTemplateSet.Namespaces[nsIndex].TemplateRef = mur.Spec.UserAccounts[uaIndex].Spec.NSTemplateSet.TierName + "-" + nsType + "-" + revision
```

Comment:
I would change it to
```suggestion
						templateRef := strings.ToLower(fmt.Sprintf("%s-%s-%s", mur.Spec.UserAccounts[uaIndex].Spec.NSTemplateSet.TierName, nsType, revision))
						mur.Spec.UserAccounts[uaIndex].Spec.NSTemplateSet.Namespaces[nsIndex].TemplateRef = templateRef
```
it's easier to read and understand.
But it's up to you ;-)

Reply:
done here: e5f1900fc95de26cbcd3d9b73a7e02ec2cb9174e